tests/logout.test.js

```diff
@@ -0,0 +1,9 @@
+describe('OpenBank', () => {
```

Comment:
You can maybe delete this file, as there is no implementation of tests yet.

Reply:
deleted...